src/LaravelLogger.php

```diff
@@ -5,9 +5,9 @@
 use Bugsnag\Client;
 use Bugsnag\PsrLogger\BugsnagLogger;
 use Illuminate\Contracts\Events\Dispatcher;
-use Illuminate\Contracts\Logging\Log;
 use Illuminate\Contracts\Support\Arrayable;
 use Illuminate\Contracts\Support\Jsonable;
+use Psr\Log\LoggerInterface as Log;
```

Comment:
This change is incorrect. The laravel interface already extends this, so this change doesn't change things for the better.

Reply:
Ah, we may need to drop older Laravel versions or make a major version bump enable this, actually.

Reply:
Hey @GrahamCampbell 👋 Can you see an alternative to allow the type hints to work on later versions? With this change I'd wondered if it might make the types not work on <5.6 but at least they'd work on modern Laravel versions?

Reply:
I'm kinda surprised that nobody has noticed this yet. The MultiLogger class is also affected.